wye/profiles/forms.py

```diff
@@ -47,3 +49,26 @@ def __init__(self, *args, **kwargs):
     class Meta:
         model = models.Profile
         exclude = ('user', 'slug')
+
+
+class ContactUsForm(forms.Form):
+
+    name = forms.CharField(label='Your name*', required=True)
+    email = forms.EmailField(label='Your email*', required=True)
+    comments = forms.CharField(label='Comments*', required=True, widget=forms.Textarea)
+    contact_number = forms.CharField(label='Your contact number', required=False)
+    feedback_type = forms.ChoiceField(label='Select Option*', required=True,
+                                      choices=ContactFeedbackType.CHOICES)
+
+    def clean_contact_number(self):
```

Comment:
@vnbang2003 : since it is an email field, so django already has built in validation for it. Hence I skipped. Let me know if it is required